controls/radwizard/getting-started.md

```diff
@@ -10,7 +10,7 @@ position: 2
 
 # Getting Started
 
-This tutorial will walk you through the creation of a sample application that contains __RadWizard__ control.
+This tutorial will walk you through the creation of a sample application that contains the __RadWizard__ control.
```

Comment:
We don't need 'the' here. Maybe we can use 'a'. The idea of this sentence is to tell that we will create an application that shows how to use a single instance of the wizard control.

"This contains the RadWizard control" is very strong saying. The Control. The Master Control. The One and Only Control. :D